dir-list-webpack-plugin.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import fs from "fs";
+
+export default class DirListWebpackPlugin {
```

Comment:
Similar to _"Pull json-webpack-plugin out into its own npm package #75"_, this plugin seems pretty awesome and probably worthy of extracting out into a dedicated npm module (in the future).

Give a 👍/:-1: and I can file a separate tracking bug for our future selves.